Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -11,11 +11,33 @@
 //
 //  Created by Nacho Soto on 2/27/22.
 
+import Foundation
+
 /// A request to be made by ``HTTPClient``
 struct HTTPRequest {
 
     let method: Method
     let path: Path
+    /// If present, this will be used by the server to compute a checksum of the response signed with a private key.
+    let nonce: Data?
+
+    init(method: Method, path: Path) {
+        self.init(method: method, path: path, nonce: nil)
+    }
+
+    init(method: Method, path: Path, nonce: Data?) {
+        assert(nonce == nil || nonce?.count == Data.nonceLength)
```

Comment:
@tonidero @aboedo thoughts on this? `assert` only compiles on `DEBUG` builds so it can help us catch issues in tests. Sending anything other than 12 bytes would fail so might as well catch it early.

Reply:
The pre-built frameworks are release, yeah. But also for people integrating with CocoaPods / SPM, their production builds are release by default.